generator/integration_tests/golden/v1/golden_kitchen_sink_client.h

```diff
@@ -19,6 +19,7 @@
 #ifndef GOOGLE_CLOUD_CPP_GENERATOR_INTEGRATION_TESTS_GOLDEN_V1_GOLDEN_KITCHEN_SINK_CLIENT_H
 #define GOOGLE_CLOUD_CPP_GENERATOR_INTEGRATION_TESTS_GOLDEN_V1_GOLDEN_KITCHEN_SINK_CLIENT_H
 
+#include "google/cloud/internal/disable_deprecation_warnings.inc"
```

Comment:
On second thought, this is not the place to push/pop. We should only push/pop around where we include the protobuf generated code.

Otherwise we are silencing the deprecation warnings for our deprecated client classes. We don't want that. We want customers to see those warnings.